### PR TITLE
frontend:Add tooltips

### LINKED ISSUE #1583: Improve UI/UX (esp. for new users)
- [x] I have searched the [issues](https://github.com/openkfw/TruBudget/issues) of this repository and believe that this is not a duplicate.

## Description 💡

List of ideas how to improve user experience/onboarding.

- [ ] add tooltips to (interactive) elements
- [ ] guided tour using intro.js, joyride, similar (minus side: has to be maintained)
- [ ] ...

### PR DESCRIPTION
**Pull request by galethil**
### Checklist

- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/openkfw/TruBudget/blob/main/.github/CONTRIBUTING.md#open-a-pull-request).
- [x] I fixed all necessary PR warnings
- [x] The commit history is clean
- [x] The E2E tests are passing
- [x] If possible, the issue has been divided into more subtasks
- [x] I did a self review before requesting a review from another team member

### Description
Add tooltips into main screens and buttons.

Closes #1583

### CODE REVIEW

File: frontend/src/pages/Workflows/Workflow.js [hunk 1]
```diff
@@ -35,17 +37,31 @@ const Workflow = (props) => {
...
-          data-test="createWorkflowitem"
-          color="primary"
-          onClick={() => props.showCreateDialog()}
-          style={{
-            position: "relative"
-          }}
+        <Tooltip
+          title={
+            !allowedToCreateWorkflows
+              ? strings.common.no_permissions
```
Comment (SamuelPull) [inline frontend/src/pages/Workflows/Workflow.js:43]: Can you move the logic  and make it more readable by using maybe nested if..else?

**Comment by SamuelPull** [comment]
Screenshot 2024-01-17 at 14 28 50
Is this tooltip on a wrong button?

**Comment by galethil** [comment]
@SamuelPull tooltip is on the correct button. It just informs the user about the reason why button is disabled. Since there could be multiple reasons why button is disabled I consider it useful for user to hover over button and see the explanation of current state.

File: frontend/src/pages/Workflows/Workflow.js (hunk 1 above)
Comment (galethil) [reply frontend/src/pages/Workflows/Workflow.js]: @SamuelPull I changed logic to if/else

**Review by SamuelPull — APPROVED** [review]

**Pull request merged**